platform/darwin/src/MGLOfflinePack.mm

```diff
@@ -93,6 +95,11 @@ - (void)resume {
 - (void)suspend {
     MGLAssertOfflinePackIsValid();
     
+    if (self.state == MGLOfflinePackStateActive) {
```

Comment:
Does `resume` need to set _isSuspending to `NO`? I'm not certain but it seems like a resumed offline pack may no longer report progress even if the state is set to `MGLOfflinePackStateActive` because of a region status change.